ui/components/app/import-token/token-list/index.scss

```diff
@@ -21,14 +19,10 @@
   }
 
   &__token_component {
-    display: flex;
-    flex-flow: row nowrap;
-    padding: 16px;
-    align-items: center;
     cursor: pointer;
 
-    &--selected {
-      background-color: var(--color-primary-muted);
+    &__network-badge {
+      box-sizing: content-box; // Prevents border effecting size of badge image
```

Comment:
Isolates box sizing just to the AvatarNetwork component in `ui/components/app/import-token/token-list/token-list.component.js`